dlgr/griduniverse/experiment.py

```diff
@@ -265,6 +268,7 @@ def __init__(self, **kwargs):
             'food_maturation_threshold', 0.0)
         self.food_planting = kwargs.get('food_planting', False)
         self.food_planting_cost = kwargs.get('food_planting_cost', 1)
+        self.food_probability_distribution = kwargs.get('food_probability_distribution', 'random')
```

Comment:
How do i add my distributions to the grid? I'm looking for the link between the `food_probability_distribution` keyword to the appropriate function